### PR TITLE
add BBoxSafeRandomCrop

### PR DESCRIPTION
**Pull request by SunQpark**
Added bbox safe random crop without implicit resizing, as suggested in Issue #480 .

I tried to update README file for this, but running `python3 tools/make_transforms_docs.py make` command only printed 
```
| Transform | Image | Masks | BBoxes | Keypoints |
| --------- | :---: | :---: | :----: | :-------: |
```

### CODE REVIEW

**Comment by Dipet** [comment]
Thank you for PR.
Maybe you do not installed you version of albumentations?

**Comment by SunQpark** [comment]
@Dipet Thank you. 
I just updated README file

File: albumentations/augmentations/transforms.py [hunk 1]
```diff
@@ -952,16 +953,11 @@ def get_transform_init_args_names(self):
         return "height", "width", "scale", "ratio", "interpolation"
 
 
-class RandomSizedBBoxSafeCrop(DualTransform):
+class BBoxSafeRandomCrop(DualTransform):
     """Crop a random part of the input and rescale it to some size without loss of bboxes.
```
Comment (IlyaOvodov) [inline albumentations/augmentations/transforms.py:957]: Update docstring here: remove "and rescale..."

Comment (SunQpark) [reply albumentations/augmentations/transforms.py]: Updated this in c0621eee7c3e03df3343c4eae5b7a571cb9cb6cb!

**Comment by tengerye** [comment]
@SunQpark @Dipet @IlyaOvodov @bmabey @henrique Hi, could you please check why this merge fails? It is 2022 now! Thank you.

**Review by Dipet — APPROVED** [review]

**Review by creafz — APPROVED** [review]

**Pull request merged**